apps/transport/lib/transport_web/templates/dataset/details.html.eex

```diff
@@ -11,7 +11,10 @@
         <%= img_tag(@dataset.full_logo, alt: @dataset.title) %>
       </div>
       <div class="dataset__title">
-        <h1><%= @dataset.title%></h1>
+        <h1>
+          <%= Dataset.nom(@dataset) %> - 
```

Comment:
shouldn't we add a `Dataset` function to have this title ? this way we can reuse it

Reply:
Yes we can!